backend/src/infra/network/outbound-url-policy.ts

```
@@ -0,0 +1,221 @@
+}
+
+function ipv4InRange(octets: number[], start: number[], end: number[]): boolean {
+  const value = octets[0] * 2 ** 24 + octets[1] * 2 ** 16 + octets[2] * 2 ** 8 + octets[3];
+  const lower = start[0] * 2 ** 24 + start[1] * 2 ** 16 + start[2] * 2 ** 8 + start[3];
+  const upper = end[0] * 2 ** 24 + end[1] * 2 ** 16 + end[2] * 2 ** 8 + end[3];
+  return value >= lower && value <= upper;
+}
+
+function isPrivateIpv4(address: string): boolean {
+  const octets = parseIpv4(address);
+  if (!octets) return false;
```

ESLint

[error] Expected { after 'if' condition.

(curly)